### PR TITLE
Add time limit

### PR DESCRIPTION
**Pull request by Marleen1**

### CODE REVIEW

File: aix360/algorithms/rbm/boolean_rule_cg.py [hunk 1]
```diff
@@ -91,6 +95,8 @@ def fit(self, X, y):
         A = np.hstack((np.ones((X.shape[0],1), dtype=int), X))
         # Iteration counter
         self.it = 0
+        # Start time
+        self.starttime - time.time()
```
Comment (dennislwei) [inline aix360/algorithms/rbm/boolean_rule_cg.py:99]: Did you mean `=` instead of `-`?

**Review by dennislwei — COMMENTED** [review]
Sorry for the delay again. Just one possible error that I noticed. 
Thanks again for doing this.

File: aix360/algorithms/rbm/boolean_rule_cg.py (hunk 1 above)
Comment (Marleen1) [reply aix360/algorithms/rbm/boolean_rule_cg.py:99]: Yes, I did, thanks for taking that out.

Comment (dennislwei) [reply aix360/algorithms/rbm/boolean_rule_cg.py]: OK I changed it.

File: aix360/algorithms/rbm/boolean_rule_cg.py [hunk 2]
```diff
@@ -126,7 +126,7 @@ def fit(self, X, y):
         v, zNew, Anew = beam_search(r, X, self.lambda0, self.lambda1,
                                     K=self.K, UB=UB, D=self.D, B=self.B, eps=self.eps)
 
-        while (v < -self.eps).any() and (self.it < self.iterMax) and (time.time()-self.starttime < timeMax):
+        while (v < -self.eps).any() and (self.it < self.iterMax) and (time.time()-self.starttime < self.timeMax):
```
Comment (dennislwei) [inline aix360/algorithms/rbm/boolean_rule_cg.py:129]: @Marleen1 I also added `self.` here in front of `timeMax` as I believe it's needed. If it looks fine to you, I will merge the pull request.

Comment (Marleen1) [reply aix360/algorithms/rbm/boolean_rule_cg.py:129]: Yes, that looks good!

**Pull request merged**